remote-agent/main.go

```diff
@@ -0,0 +1,92 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/url"
+
+	tgt "github.com/eclipse-symphony/symphony/api/pkg/apis/v1alpha1/providers/target"
+	"github.com/eclipse-symphony/symphony/api/pkg/apis/v1alpha1/providers/target/script"
+	"github.com/eclipse-symphony/symphony/remote-agent/agent"
+	"github.com/eclipse-symphony/symphony/remote-agent/bindings/http"
+	utils "github.com/eclipse-symphony/symphony/remote-agent/common"
+)
+
+func main() {
+	// Read the configuration file
+	setting, err := ioutil.ReadFile("config.json")
```

Comment:
So we expect customer needs to provide exact config file with same name under binary path? Why cannot accept -c argument?